Content.Shared/Audio/Jukebox/JukeboxPrototype.cs

```diff
@@ -20,4 +20,11 @@ public sealed partial class JukeboxPrototype : IPrototype
 
     [DataField(required: true)]
     public SoundPathSpecifier Path = default!;
+
+    /// <summary>
+    /// IMP ADDITION
+    /// True if the song should only appear when the jukebox is emagged.
+    /// </summary>
+    [DataField(required: false)]
+    public bool EmagOnly = false;
```

Comment:
bools also default to false, so this can just be `public bool EmagOnly;` and save us precious bytes